doc/reference/devicetree/api.rst

```diff
@@ -319,7 +319,7 @@ device.
      - Flash partition that the Zephyr image's text section should be linked
        into
    * - zephyr,console
-     - Sets default :kconfig:`CONFIG_UART_CONSOLE_ON_DEV_NAME`
+     - Sets UART device used by console driver
    * - zephyr,dtcm
```

Comment:
Add all the new zephyr,ot-uart, etc. here.

Reply:
> Add all the new zephyr,ot-uart, etc. here.

Still needs to be done

Reply:
See commit "net: ot: rework NCP interface configuration", https://github.com/zephyrproject-rtos/zephyr/pull/37085/commits/7d4ec15500d7b2b34fd31c03cd16d8f8e8e1f946#diff-4d66c0cb52a8c2ce6a8cf7e5f0b73fbdca77ed0f93e9f9b0d810471cd49e3ddaR342-R343